src/components/toolbar/Pagination.tsx

```diff
@@ -0,0 +1,28 @@
+import {Pagination, ToolbarItem} from '@patternfly/react-core';
+
+export function ToolbarPagination(props: ToolbarPaginationProps) {
+  return (
+    <ToolbarItem variant="pagination">
+      <Pagination
+        perPageComponent="button"
+        itemCount={props.repositoryList.length}
+        perPage={props.perPage}
+        page={props.page}
+        onSetPage={(_event, pageNumber) => props.setPage(pageNumber)}
+        onPerPageSelect={(_event, perPageNumber) => {
+          props.setPage(1);
+          props.setPerPage(perPageNumber);
+        }}
+        widgetId="pagination-options-menu-top"
+      />
+    </ToolbarItem>
+  );
+}
+
+type ToolbarPaginationProps = {
+  repositoryList: any[];
```

Comment:
nit: `repositoryList` - instead could be a generic name since this component is used for other pages as well